specification/search/data-plane/Azure.Search/preview/2020-06-30/examples/SearchIndexCountDocuments.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "endpoint": "https://myservice.search.windows.net",
+    "indexName": "myindex",
+    "api-version": "2020-06-30-Preview"
```

Comment:
This is in the 2020-06-30 directory, but has 2020-06-30-Preview as the version number. That doesn't seem right.

Remember, the `preview` and `stable` folders are orthogonal to whether the API version is suffixed with `preview` or not. It only denotes the state of the Swagger spec -- `stable` means we can't change it anymore.

Reply:
If the service and SDK are GA'ing with "2020-06-30", it shouldn't change anymore. In general, service guidelines state that swaggers should be immutable. Exceptions are often made for doc/example changes, and it's not uncommon to make non-breaking changes; though, technically, even additions can be breaking for JSON clients that don't ignore unknown model properties.

Reply:
@heaths Changes that affect codegen but don't reflect changes in the service API (like `IndexingParameters` for example) should be fine then, right?

I think a clear and unambiguous statement of what specs belong in what directory would be really helpful right now. 🙂